core/storage/topic/gae_models.py

```diff
@@ -52,7 +52,7 @@ class TopicModel(base_models.VersionedModel):
     # The canonical name of the topic, created by making `name` lowercase.
     canonical_name = ndb.StringProperty(required=True, indexed=True)
     # The abbreviated name of the topic.
-    abbreviated_name = ndb.StringProperty(required=True, indexed=True)
+    abbreviated_name = ndb.StringProperty(indexed=True)
```

Comment:
Could you please (a) add an audit job to check that all topics have a non-empty abbreviated name, and (b) file an issue to make this a required field once this job runs successfully?

Reply:
Hi @seanlip, regarding (a), should the job also populate the abbreviated name field (eg. copy over the topic name?) if it is found to be empty?

Reply:
No, it shouldn't. The flow should be that, in the release after this one, we run this just before a new deployment (and repeat until all the old abbreviated names are non-empty), and only then, do the new deployment.

Please could you make sure to update this form as well, with clear instructions, in order to ensure this is handled correctly? https://docs.google.com/forms/d/e/1FAIpQLSfvYWscAn18ok06An1oQ54h1VmBHfCX8uuuV01kIvY9WX0-Ug/viewform Alternatively you can email @ankita240796, @DubeySandeep and me with full instructions.

Thanks!

Reply:
Added an audit test for the same.